samples/xbee/demos/EndToEndAgricultureSample/main.py

```diff
@@ -1165,6 +1405,22 @@ def main():
 
     global device, current_time
 
+    # Create and configure the serial port settings
```

Comment:
Duplicated block of code, it would be better to have a method that resets the microbit board.